stable/pomerium/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,23 @@
+{{ if .Values.serviceMonitor.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ template "pomerium.fullname" . }}
+{{ if .Values.serviceMonitor.namespace -}}
+  namespace: {{ .Values.serviceMonitor.namespace }}
```

Comment:
is it standard practice to specify a different namespace than the one of the monitored service?

Reply:
Yes the operator can run in a mode where only specific namespace(s) are watched.  I don't think I've seen it used, but we should be able to support it.

Reply:
I see, thanks for the explanation

Reply:
👍